### ISSUE TITLE
Controle de Estoque

### ISSUE THREAD

**Issue by vschettino** [vschettino/lab-db#4]
### Controle de Estoque:
- **Entidades Obrigatórias:**
  - Produto
  - Fornecedor
  - Pedido
- Descrição: Projetar um sistema para controlar o estoque de uma loja, incluindo informações sobre produtos, fornecedores, pedidos e vendas.

**Comment by Igor0155** [comment]
@Igor0155 @Rodskl

**Comment by KM-10-Prog** [comment]
KM-10-Prog

**Comment by Igor0155** [comment]
![Image](https://github.com/vschettino/lab-db/assets/78672215/e650d03e-c9d5-4c2d-a474-652adcf6942c)

**Comment by Igor0155** [comment]
@vschettino  Na ultima aula eu tive que sair mais cedo, por conta disso, não pude ficar para a explicação da segunda parte dessa atividade. Poderia por gentiliza me informar se há uma segunda parte dessa atividade ?

**Comment by vschettino** [comment]
> @vschettino Na ultima aula eu tive que sair mais cedo, por conta disso, não pude ficar para a explicação da segunda parte dessa atividade. Poderia por gentiliza me informar se há uma segunda parte dessa atividade ?

Sim, vamos ter tempo de finalizar esse diagrama na aula de quinta :)
